### ISSUE TITLE
Unclear warnings during report creation

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite.ReportingEngine#970]
S. the reports here: https://github.com/Yuri05/TestReport/tree/main/Report_07_Population_Pediatric_Raltegravir (Report inputs are in the same folder)

During creation of the above reports some warnings were produced in the console and also written to the log files
(s. https://github.com/Yuri05/TestReport/blob/main/Report_07_Population_Pediatric_Raltegravir/Report/log-error.txt and https://github.com/Yuri05/TestReport/blob/main/Report_07_Population_Pediatric_Raltegravir/Report_R413/log-error.txt )

1. 
```
i Info	Starting run of Sensitivity Analysis task for Larson 2013 8y-18y 400mg FCT meal
28/12/2022 - 13:28:16                                                                                                                                       
! Warning	One or more parsing issues, see `problems()` for details
28/12/2022 - 13:28:22
...
i Info	Starting run of Plot PK Parameters task
28/12/2022 - 13:43:35                                                                                                                                       
! Warning	One or more parsing issues, see `problems()` for details
```

When I enter `problems()` after the report was created:
```
> problems()
Error in problems() : could not find function "problems"
```

2.

```
i Info	Starting run of Plot Sensitivity task
28/12/2022 - 13:43:11
! Warning	For individualID 355 : In source:
'C_max_norm' is not included in expected values: 'C_max', 't_max', 'C_tEnd', 'AUC_tEnd', 'AUC_inf', 'MRT', 'Thalf', 'FractionAucLastToInf', 'CL', 'Vss', 'Vd'.
...
! Warning	For individualID 298 : In source:
'AUC_tEnd_norm' is not included in expected values: 'C_max', 't_max', 'C_tEnd', 'AUC_tEnd', 'AUC_inf', 'MRT', 'Thalf', 'FractionAucLastToInf', 'CL', 'Vss', 'Vd'.
...
28/12/2022 - 13:43:12
! Warning	For individualID 177 : In source:
'AUC_inf_norm' is not included in expected values: 'C_max', 't_max', 'C_tEnd', 'AUC_tEnd', 'AUC_inf', 'MRT', 'Thalf', 'FractionAucLastToInf', 'CL', 'Vss', 'Vd'.
...
28/12/2022 - 13:43:15
! Warning	For individualID 66 : In source:
'AUC_tEnd_norm' is not included in expected values: 'C_max', 't_max', 'C_tEnd', 'AUC_tEnd', 'AUC_inf', 'MRT', 'Thalf', 'FractionAucLastToInf', 'CL', 'Vss', 'Vd'.
...
```

However all those parameters are available in the PK analysis results and the corresponding plots are also created
(s. e.g. https://github.com/Yuri05/TestReport/blob/9c0d2e9cc09d2826ca58bfd801d7cdb95d99d0dd/Report_07_Population_Pediatric_Raltegravir/Report_R413/PKAnalysisResults/Larson%202013%208y-18y%20400mg%20FCT%20meal-PKAnalysisResults.csv#L857 )

3.
```
i Info	Starting run of Plot PK Parameters task
28/12/2022 - 13:44:34                                                                                                                                       
! Warning	NaNs produced
28/12/2022 - 13:44:34
! Warning	NaNs produced
```

OK, here I found some NaNs in PKAnalysisResults (e.g. [here](https://github.com/Yuri05/TestReport/blob/9c0d2e9cc09d2826ca58bfd801d7cdb95d99d0dd/Report_07_Population_Pediatric_Raltegravir/Report_R413/PKAnalysisResults/Larson%202013%208y-18y%20400mg%20FCT%20meal-PKAnalysisResults.csv#L4284)) - so I assume that was the reason?

**Comment by pchelle** [comment]
Yes, the `NaN`s are provoking these errors.

The message below is actually produced by the package `{readr}` when called by the function `ospsuite::pkAnalysesToDataFrame` (`problems()` is a function from the `{readr}` package). 
```
One or more parsing issues, see `problems()` for details
```  

To transform the PK Analysis object into a data.frame, the function is exporting a temporary csv file read by the function [`readr::read_csv` with specifications about the column types](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/91392e1f915d4accb1e1e0045f4d191089af3e10/R/utilities-pk-analysis.R#L81).

The problem seems that the specification `col_double` for the reading of the column `"Value"` is not happy with `NA`s, `NaN`s or/and `Inf`s values. From what I tested, 
* `Inf` are not recognized as double, throwing a warning and translated as `NA`
* `NA` and `""` are recognized (through default option `na=c("", "NA")`), and translated as `NaN` as the column is assumed double

Oddly, parsing as character (`col_character`) and then using `as.numeric` on the result seems to appropriately keep the information that infinite values are infinite.

**Comment by pchelle** [comment]
Fixed by PR [#1203](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/pull/1203)
